scripts/presubmit.sh

```diff
@@ -132,7 +132,7 @@ main() {
     done | xargs -I '{}' -P ${GO_TEST_PARALLELISM:-10} bash -c '{}'
 
     [[ ${coverage} -eq 1 ]] && \
-      cat /tmp/ct_profile/*.out > /tmp/coverage.txt
+      cat /tmp/ct_profile/*.out > coverage.txt
```

Comment:
Seems better to write this to a fixed place but I assume there's a reason for changing that.

Reply:
yeah, codecov no longer picked it up because '/tmp' is transient (we're running in docker containers on CB) so I'm dropping this inside the repo which works since it's in the shared workspace volume.